src/node_modules/app/tsservices/index.js

```diff
@@ -22,12 +22,15 @@ export function getCompletions (
 		);
 		return {
 			name: completionData.name,
-			required: details.displayParts[5].text !== '?',
-			displayText:
-				details.displayParts.slice(7)
-				.map(part => part.text)
-				.join('')
-				.trim(),
+			required: details.displayParts.length > 5
+				? details.displayParts[5].text !== '?'
+				: false,
+			displayText: (details.displayParts.length > 5
+				? details.displayParts.slice(7)
+				: details.displayParts)
+			.map(part => part.text)
+			.join('')
+			.trim(),
```

Comment:
this needs to be indented

Reply:
Linting flags indentation as incorrect.

Reply:
I reworked the formatting to make it clearer. Uploading a screen capture for speed:
![image](https://user-images.githubusercontent.com/1522062/97498141-69e7b800-194a-11eb-8921-2335836d683f.png)